apis/nucleus/src/sn/type.js

```diff
@@ -1,6 +1,6 @@
 import { generator as SNFactory } from '@nebula.js/supernova';
-import { satisfies } from 'semver';
 import extend from 'extend';
+import satisfies from 'semver/functions/satisfies';
```

Comment:
This was an attempt to solve the circular dependency warning detailed here: https://github.com/npm/node-semver/issues/381
Still get the warning however.